app/models/concerns/thredded/post_common.rb

```diff
@@ -13,14 +13,18 @@ module PostCommon
 
       validates :content, presence: true
 
-      scope :order_oldest_first, -> { order(id: :asc) }
-      scope :order_newest_first, -> { order(id: :desc) }
+      scope :order_oldest_first, -> { order(created_at: :asc) }
```

Comment:
Ugh, really? -- millisecond accuracy in postgres/sqlite. and (ok) second accuracy in mysql. How many times are these *actually* going to clash? And would it be surprising to anyone if the ordering was indeterminate at this point (and how many times are they going to clash at a page boundary. I'm going to go out on a limb and say 'never').